plugins/wallabag.koplugin/main.lua

```diff
@@ -188,12 +188,12 @@ function Wallabag:addToMainMenu(menu_items)
                     {
                         text_func = function()
                             local tags
-                            if not self.ignore_tags or self.ignore_tags == "" then
-                                tags = _("None")
+                            if not self.ignore_tags then
+                                tags = ""
```

Comment:
But then it'll just show `Ignore tags ()`. That's ugly. :-) My `return _("Ignore tags")` suggestion doesn't have that problem.

The problem with the `Ignore tags: %1` string is that you're programmatically combining the word `none` with a string. To translate the word none correctly, chances are it might need to be a contextualized (see [here](http://koreader.rocks/doc/modules/gettext.html#pgettext)). That's a lot of extra mental time and effort for a translator to wrap their head around when the full string `Ignore tags: none` will do the job just fine. Not to mention that a user might have a `none` tag (meaning something like "doesn't fit the other categories).

*However*, that's all a moot point because I think a simple `Ignore tags` is better overall. It's just a PSA: don't be clever with strings that have to be localized. Programmers always want to DRY (for good reason of course), but for localization it's often better to repeat yourself a little.

What works in English will likely to work quite well in closely related languages like Dutch and German, not typically too bad either in mostly similar but more distantly related languages like French. But as soon as you leave Western Europe, programmatic assumptions about language will bite you (or perhaps more to the point, they'll make all of our hard-working translators sad). It's as simple as that. ;-)

Reply:
So in short, I stick with:
```suggestion
                                return  _("Ignore tags")
```

Reply:
I was wondering what those `_`, `T("foo")` functions were about, thanks for linking that documentation. 

I think I got it, unless what you wanted was just one `return _("Ignore tags")`? Because if I understand this correctly, if someone localizes `"Ignore tags"`, maybe someone else could localize `"Ignore tags (%1)"` differently right?

Reply:
Right, that's the translator's prerogative. Even in English it might make sense to make the strings slightly different, like:

* `Ignore tags`
* `Ignored tags (one,two,three)`

Incidentally, I didn't notice this before merging but `Help` is now relegated to the second page.

![Screenshot_20191013_091136](https://user-images.githubusercontent.com/202757/66712209-7e85fc00-ed99-11e9-8c66-698760e0c74c.png)

It might make sense to group these two together in a `Tags` submenu.

Reply:
That would also open it up to more easily splitting the ignored tags themselves off from the ignoring. See for example how the multiswipe settings are grayed out when multiswipes aren't enabled, but you don't have to empty your settings to disable them. (Although if you're ignoring more than a handful of tags, you should probably just filter them? Or tag less? lol)

![Screenshot_20191013_092208](https://user-images.githubusercontent.com/202757/66712308-0c161b80-ed9b-11e9-916b-da6f03e877ec.png)

Anyway, just some thoughts. Thanks for the addition!